src/OrleansSQLUtils/OrleansSQLUtils.csproj

```diff
@@ -1,115 +1,41 @@
-    <Compile Include="Storage\AdoNetConfigurationExtensions.cs" />
-    <Compile Include="Storage\AdoNetInvariants.cs" />
-    <Compile Include="Storage\DbConstantsStore.cs" />
-    <Compile Include="Storage\DbExtensions.cs" />
-    <Compile Include="Storage\DbConnectionFactory.cs" />
-    <Compile Include="Storage\IRelationalStorage.cs" />
-    <Compile Include="Storage\Provider\AdoGrainKey.cs" />
-    <Compile Include="Storage\Provider\IHasher.cs" />
-    <Compile Include="Storage\Provider\IStorageDeserializer.cs" />
-    <Compile Include="Storage\Provider\IStorageHashPicker.cs" />
-    <Compile Include="Storage\Provider\StorageHasherPicker.cs" />
-    <Compile Include="Storage\Provider\IStorageSerializationPicker.cs" />
-    <Compile Include="Storage\Provider\IStorageSerializer.cs" />
-    <Compile Include="Storage\Provider\OrleansDefaultHasher.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultBinarySerializer.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultBinaryDeserializer.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultJsonDeserializer.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultJsonSerializer.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultXmlDeserializer.cs" />
-    <Compile Include="Storage\Provider\OrleansStorageDefaultXmlSerializer.cs" />
-    <Compile Include="Storage\Provider\AdoNetStorageProvider.cs" />
-    <Compile Include="Storage\Provider\RelationalStorageProviderQueries.cs" />
-    <Compile Include="Storage\Provider\StorageSerializationPicker.cs" />
-    <Compile Include="Storage\RelationalStorage.cs" />
-    <Compile Include="Storage\RelationalOrleansQueries.cs" />
-    <Compile Include="Storage\RelationalStorageExtensions.cs" />
-    <Compile Include="Storage\SqlFormatProvider.cs" />
-    <Compile Include="Storage\SqlStatisticsPublisher.cs" />
-    <Compile Include="Storage\DbStoredQueries.cs" />
+    <ProjectReference Include="..\Orleans\Orleans.csproj" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Orleans\Orleans.csproj">
-      <Project>{bc1bd60c-e7d8-4452-a21c-290aec8e2e74}</Project>
-      <Name>Orleans</Name>
-    </ProjectReference>
+
+  <ItemGroup Condition=" '$(BuildFlavor)' == 'Current' ">
+    <PackageReference Include="System.Data.Common" Version="$(SystemRuntimeVersion)" />
+    <PackageReference Include="System.Reflection" Version="$(SystemRuntimeVersion)" />
+    <PackageReference Include="System.Xml.XmlSerializer" Version="$(SystemRuntimeVersion)" />
   </ItemGroup>
+
   <ItemGroup>
-    <Content Include="CreateOrleansTables_MySql.sql">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="CreateOrleansTables_PostgreSql.sql">
+    <None Include="CreateOrleansTables_MySQL.sql">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Pack>true</Pack>
+      <PackagePath>lib\$(TargetFramework)\MySQL\</PackagePath>
```

Comment:
Why in lib\$(TargetFramework)? Is this how it was done before and you are just preserving the location?
Otherwise it might make sense to put them in some other not-framework specific folder. But if it's preserving it as it was before, ignore and keep it like this for now